### DISCUSSION TITLE
Alignment of braces in nested initialization

### DISCUSSION THREAD

**Discussion by liu15** [uncrustify/uncrustify#4436] [Q&A]
I am trying to update from uncrustify version 0.63 to the latest version and I'm having trouble identifying the proper controls (if they exist).

I have the following bit of code:

```
//                c  +  -  
int a[4][3] = { { 0, 1, 3 },  
                { 1, 2, 0 },  
                { 2, 3, 1 },  
                { 3, 0, 2 } };  
```

and I would like the existing alignment to be preserved, rather that indenting the second line at the tab stop as follows:
```
//                c  +  -  
int a[4][3] = { { 0, 1, 3 },  
   { 1, 2, 0 },  
   { 2, 3, 1 },  
   { 3, 0, 2 } };  
```

I am using the following config:

indent_with_tabs = 0
input_tab_size = 3
output_tab_size = 3
indent_columns = output_tab_size
indent_ignore_first_continue = true

Is it possible to prevent this indentation?

**Comment by gmaurel** [comment]
it is a known problem. No solution now.
